tests/app/dao/test_ft_billing_dao.py

```diff
@@ -606,62 +606,111 @@ def test_fetch_sms_billing_for_all_services_with_remainder(notify_db_session):
 
 
 def test_fetch_sms_billing_for_all_services_without_an_organisation_appears(notify_db_session):
-    org, org_2, service, service_2, service_3, service_sms_only, \
-        org_with_emails, service_with_emails = set_up_usage_data(datetime(2019, 5, 1))
+    setup = set_up_usage_data(datetime(2019, 5, 1))
```

Comment:
I was thinking and could not get the right word. `services` is more descriptive, but as you say, it is not correct, and changing the fixture just for the name to fit feels like doing things wrong way round. 

Also I did not want to make the name too long as that would also impact readability, with the long service and org names. In the end I thought since in the results we always call out the key names which are very descriptive, the name of the dict is less important as long as it's short and sweet.

Reply:
> changing the fixture just for the name to fit feels like doing things wrong way round

How would it be the wrong way round?

I realise you've done some work already to make this fixture a bit better, so I'm OK with keeping it as-is. I don't think `setup` is a good name, though, as it's not a term we use elsewhere and it also means other things in Python. If you'd like to keep the fixture as is, then I'd suggest a name like `fixtures` is more descriptive for what the variable contains.

> I was wondering if we really need one method, or if it could be split into separate fixtures?

☝️ not sure if you saw this in the main comment. I think this fixture shouldn't exist, as it does way too many things. But it would also be more work to unpick it for the different tests, so maybe something for the future.

Reply:
I can change name to fixtures, the rest I feel is out of scope for this PR ☺️